lizmap/modules/admin_api/controllers/rest.classic.php

```diff
@@ -0,0 +1,83 @@
+<?php
+/**
+ * @package   lizmap
+ * @subpackage admin_api
+ * @author    3liz.com
+ * @copyright 2011-2025 3Liz
+ * @link      https://3liz.com
+ * @license   https://www.mozilla.org/MPL/ Mozilla Public Licence
+ */
+
+use LizmapAdmin\RepositoryRightsService;
+use LizmapApi\RestApi;
+use LizmapApi\Credentials;
+use LizmapApi\Utils;
+use LizmapApi\Error;
+
+class restCtrl extends RestApi {
```

Comment:
Rename the file `repository_rest` and the class `repository_restCtrl`, and update the urls.xml